src/common-principles.md

```diff
@@ -536,12 +554,40 @@ like in the example below.
             "F": {
                 "Description": "Female",
                 "TermURL": "https://www.ncbi.nlm.nih.gov/mesh/68005260"
-            },
+            }
         }
     }
 }
 ```
 
+### Compressed tabular files
+
+Large tabular information, such as physiological recordings, SHOULD be stored with
```

Comment:
As phrased here, this would be a kind of meta-requirement on spec authors to use TSVGZ if they're large, not a requirement on curators/converters, who will receive direction from the individual data type specifications. The problem here is that motion has violated this, using yet another TSV + `channels.tsv` configuration.

Reply:
You mean, it would be better to roll back to SHOULD? Happy to change as needed if the problem is not the particular term.

Reply:
Actually, reading more context, it's ` MUST be stored [...] when so established by the specifications.`

That's fine then.